src/transcript/TranscriptEvent.ts

```diff
@@ -107,6 +116,24 @@ export class TranscriptEventConverter {
               transcriptAlternative.items.push(transcriptItem);
             }
 
+            for (const entity of alternative.entities) {
+              if (!transcriptAlternative.entities) {
+                transcriptAlternative.entities = [];
+              }
+              const transcriptEntity: TranscriptEntity = {
+                category: entity.category,
+                confidence: entity.confidence,
+                content: entity.content,
+                startTimeMs: entity.startTime as number,
+                endTimeMs: entity.endTime as number,
```

Comment:
Are `entity.startTime` and `endTime` numbers in the input? The `as` won't convert, it's just a TypeScript cast. If these come in as strings, you need `Number.parseInt(x, 10)`.

Reply:
startTime is an int64 field which gets converted to number|long|null.
endTime?: (number|Long|null);

Reply:
I think you're still missing the point of my question. `entity` is a JavaScript object. `entity.startTime` is an attribute with some value. If that's always a `number`, you don't need `as number`. If it's not always a `number`, you need some more sophisticated transformation here. So why do you need the `as`?

Reply:
So long as it's guarded by tests, that's fine.

Reply:
Got it. We do have tests for this now.